test/Generator/ClassGeneratorTest.php

```diff
@@ -563,16 +564,69 @@ public function testAddConstantThrowsExceptionWithInvalidName()
         $classGenerator->addConstant([], 'value1');
     }
 
-    /**
-     * @group 6274
-     */
-    public function testAddConstantThrowsExceptionWithInvalidValue()
+    public function testAddConstantThrowsExceptionWithEmptyConstantName()
     {
-        $this->setExpectedException('InvalidArgumentException');
+        $classGenerator = new ClassGenerator();
+
+        $this->setExpectedException(InvalidArgumentException::class);
+
+        $classGenerator->addConstant('', 'value');
+    }
+
+    public function testAddConstantAcceptsMixedScalars()
+    {
+        $classGenerator = new ClassGenerator();
+
```

Comment:
I would like examples for arithmetic expressions. 

2 + - / * ** % 2  + Another::COSTANT